research-software-directory/RSD-as-a-service
# Bulk import / Import publications

# Import publications (software and projects)

Changes proposed in this pull request:
* Bulk import button is renamed to Import publications section and set at the second position
* This section is used on edit software/mentions, project/input and project/output pages
* The max number of items is set to 50 and the validation on row count is implemented
* Wait cursor during validation is implemented
* Validation on already included publication is implemented
* Info text on software/mention, project/impact en project/output pages are updated.

How to test:
* `make start` to rebuild
* use DOI input examples from #808 
* Try to import same list twice in the same software/project. You should get respose from validation report that the entries are already in the software/project
* Validate text changes in info boxes

## Position of section and the information (Software mentions)
![image](https://user-images.githubusercontent.com/9204081/229903521-1c66e95c-856f-4b7d-8c12-99df0ca3caaf.png)

## Position of section and the information (Project impact)
![image](https://user-images.githubusercontent.com/9204081/229903154-f169064c-380a-4e51-ab04-b44cd869ca03.png)

## Position of section and the information (Project output)
![image](https://user-images.githubusercontent.com/9204081/229903252-1192564d-ab96-4b04-8545-39af6638475b.png)

## Input DOI modal
![image](https://user-images.githubusercontent.com/9204081/229739123-d25f1d73-5ba0-4c76-9efd-772f46ef0ac8.png)

## Selection and validation modal
![image](https://user-images.githubusercontent.com/9204081/229740041-67aff7d2-6b5a-40b7-b7d6-315305ba044f.png)

## New manual item
![image](https://user-images.githubusercontent.com/9204081/229904195-63149803-ebfb-47a6-9560-b814ce4dcd17.png)


PR Checklist:
